hummingbot/connector/gateway_EVM_AMM.py

```diff
@@ -613,9 +614,78 @@ async def _update_balances(self, on_interval=False):
             self._in_flight_orders_snapshot = {k: copy.copy(v) for k, v in self._in_flight_orders.items()}
             self._in_flight_orders_snapshot_timestamp = self.current_timestamp
 
+    async def execute_cancel(self, trading_pair: str, client_order_id: str) -> str:
+        await self._general_cancel(client_order_id, None)
+
+    async def _general_cancel(self, order_id: str, cancel_age: Optional[int] = None) -> str:
```

Comment:
If this function is not expected to always return a `str` - then its return type should be `Optional[str]`.